test/e2e/ha/openstack_services_test.go

```diff
@@ -289,6 +291,13 @@ func assertOpenStackReplicasReady(t *testing.T, w wait.Wait, r int32) {
 	})
 }
 
+func assertPostgresReady(t *testing.T, w wait.Wait, r int32) {
```

Comment:
since it is not an openstack component

Reply:
yeah so maybe you can change this assertOpenstack method to something like assertServices ...
I would suggest having one method since you won't forget about some case.

Reply:
okay